thumbnails/CHANGELOG.md

```diff
@@ -0,0 +1,147 @@
+# Changelog for [0.3.0] (2020-08-17)
+
+The following sections list the changes in ocis-thumbnails 0.3.0.
+
+[0.3.0]: https://github.com/owncloud/ocis/thumbnails/compare/v0.2.0...v0.3.0
+
+## Summary
+
+* Bugfix - Build docker images with alpine:latest instead of alpine:edge: [#35](https://github.com/owncloud/ocis/thumbnails/pull/35)
+* Enhancement - Serve the metrics endpoint: [#37](https://github.com/owncloud/ocis/thumbnails/issues/37)
+
+## Details
+
+* Bugfix - Build docker images with alpine:latest instead of alpine:edge: [#35](https://github.com/owncloud/ocis/thumbnails/pull/35)
+
+   ARM builds were failing when built on alpine:edge, so we switched to alpine:latest instead.
+
+   https://github.com/owncloud/ocis/thumbnails/pull/35
+
+
+* Enhancement - Serve the metrics endpoint: [#37](https://github.com/owncloud/ocis/thumbnails/issues/37)
+
+   Added the metrics endpoint to be able to monitor this service like the other services
+
+   https://github.com/owncloud/ocis/thumbnails/issues/37
+
+# Changelog for [0.2.0] (2020-07-10)
+
+The following sections list the changes in ocis-thumbnails 0.2.0.
+
+[0.2.0]: https://github.com/owncloud/ocis/thumbnails/compare/v0.1.2...v0.2.0
+
+## Summary
+
+* Change - Add more default resolutions: [#23](https://github.com/owncloud/ocis/thumbnails/issues/23)
+* Change - Refactor code to remove code smells: [#21](https://github.com/owncloud/ocis/thumbnails/issues/21)
+* Change - Use micro service error api: [#31](https://github.com/owncloud/ocis/thumbnails/issues/31)
+* Enhancement - Limit users to access own thumbnails: [#5](https://github.com/owncloud/ocis/thumbnails/issues/5)
+
+## Details
```

Comment:
Codacy found an issue: [[no-duplicate-headings] Do not use headings with similar content (12:1)](https://app.codacy.com/gh/owncloud/ocis/pullRequest?prid=6202733)